src/utils/query-params.ts

```diff
@@ -0,0 +1,15 @@
+export class QueryParams {
+  readonly params;
+  constructor(query) {
+    this.params = query;
+  }
+  get q() {
+    return this.params.q;
+  }
+  get something() {
+    return this.params.something;
+  }
+  getParamByName(param: string) {
+    return this.params[param];
+  }
```

Comment:
1. Есть смысл в дженерике
2. Без интерфейса params неясно что можно там вообще быть

```
  getParamByName<T>(param: string): T {
     return <T>this.params[param];
   }
```

Reply:
1. Пока не реализовано. Создал задачу NSB-9
2. 

```suggestion
  getParamByName(name: ParamsList | ParamsSingle): string {
    return this.params[name];
  }
```